mkdocs/topics/sources.md

```diff
@@ -2,7 +2,7 @@
 Sources are data sources. They are databases that contain your data. This page describes some best practices how to set sources up. 
 
 !!! tip
-    There is specific documentation on how to connect to a variety of databases in the Configuration documentation ([ie. PostgresSQL](/../reference/configuration/Sources/PostgresqlSource/))
+    There is specific documentation on how to connect to a variety of databases in the Configuration documentation ([ie. PostgresSQL](/../reference/configuration/targets/PostgresqlSource/))
```

Comment:
Think this should be sources

Reply:
It should be sources. For some reason our parser is still labeling it as targets. I think maybe its coming from the aliasing? I didn't want to spend too much time digging into it, but it does bother me too

Reply:
Any chance it is the capitalization?